save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/Execution.kt

```diff
@@ -71,6 +76,10 @@ class Execution(
     @JoinColumn(name = "user_id")
     var user: User?,
 
+    var execCmd: String?,
```

Comment:
In TestResourceSelection these fields are not nullable

Reply:
But they are nullable in the DB